Content.Shared/Shipyard/Components/ShuttleDeedComponent.cs

```diff
@@ -8,12 +8,12 @@ namespace Content.Shared.Shipyard.Components;
 [RegisterComponent, Access(typeof(SharedShipyardSystem))]
 public sealed partial class ShuttleDeedComponent : Component
 {
-    [DataField("shuttleUid")]
+    [ViewVariables(VVAccess.ReadWrite), DataField("shuttleUid")]
     public EntityUid? ShuttleUid;
 
-    [DataField("shuttleName")]
-    public string? ShuttleName;
+    [ViewVariables(VVAccess.ReadWrite), DataField("shuttleName")]
+    public string? ShuttleName = "Unknown";
 
-    [DataField("shuttleOwner")]
-    public EntityUid? ShuttleOwner;
+    [ViewVariables(VVAccess.ReadWrite), DataField("shuttleOwner")]
+    public EntityUid? ShuttleOwner = null;
```

Comment:
why? I'd like to restrict access to components as much as possible, and i dont understand why you would want to allow these to be modified.

Reply:
Asked by Quigley to be added, removing.